lib/app.js

```diff
@@ -58,6 +58,28 @@ app.get('./location', async (req, res) => {
   }
 });
 
+app.get('./weather', async (req, res) =>{
```

Comment:
Your endpoints here should only be `/endpoint`, they don't have a `./` like file navigation in your repo since these are part of the actual URL that a user will be visiting. Currently, this is breaking all of your endpoints so I'm not able to load any of them.